### PR TITLE
Update zha.markdown ZHA quirks handling

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Update zha.markdown ZHA quirks handling.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Hedda** [comment]
Not stale

**Comment by frenck** [comment]
Are you planning on moving it forward then?

../Frenck

**Comment by Hedda** [comment]
> Are you planning on moving it forward then?
> 
> ../Frenck

How can and should I move it forward? Is not the missing review the only blocker? Is not reviewers required if to move it forward?

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -426,11 +426,13 @@ Tip to new users is that, while there is no official list of supported devices,
...
 
-The ZHA implementation in Home Assistant relies on a library called "[ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)" to resolve issues with Zigbee devices that do not fully conform with the Zigbee standards. The few devices that deviate from the Zigbee specifications set by the [Zigbee Alliance](https://zigbeealliance.org) may therefore require proper bug reports with debug logs from users to assistant the developers in writing custom [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) for all of a device functions to work properly with the ZHA integration.
+Home Assistant's ZHA integration already supports most standard device types out-of-the-box as long as they follow the official Zigbee specifications, but for devices that manufacturers have not properly not fully Zigbee compatible the ZHA integration has implemented a library called "[ZHA Device Handlers (also known as "zha-quirk")](https://github.com/zigpy/zha-device-handlers)" that handle and resolve compliance issues via custom conversion/translation of Zigbee parameters (clusters and attributes) for specific devices which do not conform with the Zigbee standards.
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
Home Assistant's ZHA integration already supports most standard device types out-of-the-box as long as they follow the official Zigbee specifications, but for devices that manufacturers have not properly not fully Zigbee compatible, the ZHA integration has implemented a library called "[ZHA Device Handlers (also known as "zha-quirk")](https://github.com/zigpy/zha-device-handlers)" that handle and resolve compliance issues via custom conversion/translation of Zigbee parameters (clusters and attributes) for specific devices which do not conform with the Zigbee standards.
```

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -426,11 +426,13 @@ Tip to new users is that, while there is no official list of supported devices,
...
+The few devices that will for example not join/pair properly or while joined/paired do not present all attributes in the ZHA integration likely deviate from the Zigbee specifications set by the [CSA (Connectivity Standards Alliance, formerly the Zigbee Alliance)](https://csa-iot.org/all-solutions/zigbee/) and may therefore require proper bug reports with debug logs from the end-users and as a device owner you may need to actively assistant troubleshooting by providing the information developers need to code custom [ZHA Device Handlers)](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) that convert/translate non-compliant device functions so they will work properly with the ZHA integration.
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
The few devices that will, for example, not join/pair properly or, while joined/paired, do not present all attributes in the ZHA integration likely deviate from the Zigbee specifications set by the [CSA (Connectivity Standards Alliance)](https://csa-iot.org/all-solutions/zigbee/). These devices may therefore require the creation of a bug report by a device owner, supplying debug logs. The device owner may need to actively assist in troubleshooting by providing the information developers need to create custom ZHA Device Handlers.
```

File: source/_integrations/zha.markdown [hunk 3]
```diff
@@ -426,11 +426,13 @@ Tip to new users is that, while there is no official list of supported devices,
...
+Such a custom "ZHA Device Handler" are Python script that internally is also referred to as a "quirk" because they fix or workaround "quirks" in devices, like deviations from the standard specifications. ZHA Device Handles do this by transparently, acting as an in-line translator/converter, translating and converting non-compliant device messages and instead presenting them to the application as coming from a virtual compliant device that follows the official Zigbee standard. These ZHA Device Handlers for Home Assistant can thus be used to parse custom messages to and from Zigbee devices. Once a quirk is coded, the ZHA Device Handlers that are made can then be reused by all users with the same devices in future versions of Home Assistant.
```
Comment (frenck) [inline source/_integrations/zha.markdown]: This whole chapter is repeating all the above, in different wording, and can thus be removed without losing information.

```suggestion
```

File: source/_integrations/zha.markdown [hunk 4]
```diff
@@ -426,11 +426,13 @@ Tip to new users is that, while there is no official list of supported devices,
...
+
+For reference, the ZHA integration's custom quirks implementations for zigpy implemented as ZHA Device Handlers for Home Assistant are a similar concept to that of other Zigbee gateway/hub implementations, such as [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well as that of [Zigbee-Herdsman Converters (formerly Zigbee-Shepherd Converters) as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning each of those uses virtual representations of physical devices that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms.
```
Comment (frenck) [inline source/_integrations/zha.markdown]: Non of this is relevant to an end-user.
Let's remove it (above all, it is not even about ZHA).

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @Hedda 👍 

../Frenck

**Pull request merged**